thirdparty_containers/tomcat/build.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<project name="tomcat-test" default="build" basedir=".">
+	<taskdef resource="net/sf/antcontrib/antlib.xml" />
+	<description>
+		Build tomcat-test Docker image
+	</description>
+
+	<!-- set properties for this build -->
+	<property name="DEST" value="${BUILD_ROOT}/thirdparty_containers/tomcat-test" />
+	<property name="dist" location="${DEST}/${JAVA_VERSION}" />
+	<property name="src" location="." />
+	<property name="jvm_version" location="${JVM_VERSION}" />
+
+	<target name="init">
+		<mkdir dir="${dist}"/>
```

Comment:
Change to ```<mkdir dir="${DEST}"/>```

Reply:
I have made these changes, please review.